src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -622,69 +620,95 @@ private AIState waitForShovel()
     }
 
     /**
-     * Ensures that we have a shovel available.
-     * Will set {@code needsShovel} accordingly.
+     * Wait for a needed shovel.
      *
-     * @return true if we have a shovel
+     * @return NEEDS_SHOVEL
      */
-    private boolean checkForShovel()
+    /*@NotNull
+    private AIState waitForTool()
     {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)